draft-ietf-add-ddr.md

```diff
@@ -214,20 +237,20 @@ certificate can be validated, the client SHOULD use the discovered Designated
 Resolver for any cases in which it would have otherwise used the
 Unencrypted Resolver. If the Designated Resolver has a different IP
 address than the Unencrypted Resolver and the TLS certificate does not cover the
-Unencrypted Resolver address, the client MUST NOT use the discovered Encrypted
-Resolver. Additionally, the client SHOULD suppress any further queries for
+Unencrypted Resolver address, the client MUST NOT automatically use the discovered
+Designated Resolver. Additionally, the client SHOULD suppress any further queries for
 Designated Resolvers using this Unencrypted Resolver for the length of
 time indicated by the SVCB record's Time to Live (TTL).
 
 If the Designated Resolver and the Unencrypted Resolver share an IP
 address, clients MAY choose to opportunistically use the Encrypted Resolver even
```

Comment:
[not blocking] I think all of the instances of "Encrypted" versus "Designated" make sense except this one. What do you think of replacing "Encrypted Resolver" with "Designated Resolver" for consistency within this sentence?

Reply:
Fixed up!